include/lug/Window/Event.hpp

```diff
@@ -1,17 +1,55 @@
 #pragma once
 
 #include <lug/Window/Export.hpp>
+#include <lug/Window/Keyboard.hpp>
+
+#include <cstdint>
```

Comment:
Order of includes (or we can do the meeting of the Guidelines as soon as possible)